streaming/multimodal/convert/webvid/README.md

```diff
@@ -1,13 +1,43 @@
 # WebVid
 
-## crawl.py
+[WebVid](https://m-bain.github.io/webvid-dataset/) is a large-scale dataset of short videos with textual descriptions sourced from stock footage sites. There are two sets of the dataset available:
+1. 10M: It contains 10 million video-text pairs.
+2. 2.5M: It contains 2.5 million video-text pairs.
+
+## Dataset preparation
+
+To use Streaming Dataset, we must first convert the dataset from its native format to MosaicML's Streaming Dataset format called Mosaic Dataset Shard (MDS). Once in MDS format, we can access the dataset from the local file system (disk network attached storage, etc.) or object store (GCS, OCS, S3, etc.). From the object store, data can be streamed to train deep learning models, and it all works efficiently.
+
+Check out the steps below for information on converting WebVid datasets to MDS format—also, checkout [MDSWriter()](https://streaming.docs.mosaicml.com/en/latest/api_reference/generated/streaming.MDSWriter.html) parameters for details on advanced usage.
+
+#### Single MDS dataset conversion
 
 Create an MDS dataset from a CSV file containing video URLs (downloads the videos).
 
-## crawl_subsets.py
+1. Navigate to the [WebVid download section](https://m-bain.github.io/webvid-dataset/), where you will find 2.5M and 10M dataset splits. Download each CSV split you want to process.
+2. Run the [webvid.py](https://github.com/mosaicml/streaming/blob/main/streaming/multimodal/convert/webvid/webvid.py) script with a minimum required arguments as shown below  
```

Comment:
Why the rename?

Reply:
Following the same notion as what we have in `text/` and `vision/` by having a dataset name in it since we also render the readme to other places to ensure there is no confusion.

Reply:
Isn’t this covered by us having the directory called webvid/?

The other scripts were named in that convention because there was only one of them per dataset

Here, surely we should distinguish them by naming them after what they do, which is crawling samples, especially because crawling generates a ton of traffic (also inside to outside was not renamed?)

Reply:
Maybe like this?

`crawl_webvid.py`
`extract_webvid_videos.py`
`crawl_webvid_subsets.py`